index.bs

```diff
@@ -670,6 +686,8 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
     </div>
     </li>
 
+    <li><dfn>Focus distance</dfn> is a numeric camera setting that controls the focus distance of the lens.  The setting usually represents distance in millimeters to the optimal focus distance.</li>
```

Comment:
I don't think there's anything that insists its millimeters, just that it is a logical measurement to use. I believe manufacturers can make it arbitrary if they'd like. The Android API doesn't mention units at all, just the term distance.